sycl/unittests/queue/wait.cpp

```diff
@@ -0,0 +1,84 @@
+#include <CL/sycl.hpp>
```

Comment:
I believe other unit tests have some header in sources.

Reply:
Done